### DISCUSSION TITLE
Enemy: Littoral Cancer

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/Prophets-of-the-Labyrinth#278] [Ideas]
Pun: littoral = shore (military context?), cancer = crab
Labyrinth: Zoo of Chimeras
Element: Water? (maybe pick something else if it becomes a final battle, since Elkemist also has Water)
Brainstorm Credit: AwkwardLion

**Comment by doubledh** [comment]
implies ads

**Reply by ntseng** [reply]
Not opposed, but hadn't imagined that myself. Did you have something in mind?
